templates/default/index.php

```diff
@@ -79,7 +81,7 @@
 	<meta name="theme-color" content="<?= $themeColor ?>">
 	<?php endif; ?>
 </head>
-<body data-bs-theme="<?= $darkModeValue ?: '' ?>">
+<body data-bs-theme="<?= $darkModeValue ?: '' ?>" class="<?= $view ?: '' ?>">
```

Comment:
Change that to `class="panopticon-view-<?= $view ?>"` to avoid naming clashes. For example, the main view now adds `panopticon-view-main`.